lib/kino/render.ex

```diff
@@ -90,6 +90,18 @@ defimpl Kino.Render, for: Kino.Control do
   end
 end
 
+defimpl Kino.Render, for: Audio do
+  def to_livebook(audio) do
+    Kino.Output.audio(audio.content, audio.mime_type)
+  end
+end
+
+defimpl Kino.Render, for: Video do
+  def to_livebook(video) do
+    Kino.Output.video(video.content, video.mime_type)
+  end
+end
```

Comment:
Please let's wait for @jonatanklosko to confirm but I am almost sure you don't need this if using Kino.JS. Although it is unclear if we should use Kino.JS or handle it in Livebook. Why is Kino.Image handled in Livebook again?

Reply:
> I am almost sure you don't need this if using Kino.JS

Yeah, the changes in `render.ex` and `output.ex` are not necessary :)

> Although it is unclear if we should use Kino.JS or handle it in Livebook.

I would do it using Kino APIs, because in the future we can extend it with streaming (`Kino.JS.Live` should have all the APIs we need) and this stays transparent to Livebook.

> Why is Kino.Image handled in Livebook again?

IIRC our reasoning was that it's common enough to justify special casing and not using an iframe.